lib/ddtrace/tracer.rb

```diff
@@ -210,8 +208,8 @@ def start_span(name, options = {})
         # child span
         span.parent = parent # sets service, trace_id, parent_id, sampled
       end
-      tags.each { |k, v| span.set_tag(k, v) } unless tags.empty?
       @tags.each { |k, v| span.set_tag(k, v) } unless @tags.empty?
+      tags.each { |k, v| span.set_tag(k, v) } unless tags.empty?
```

Comment:
This fixes the other bug where default tags were overriding tags manually set upon the span.